lib/moment-range.js

```diff
@@ -36,31 +36,24 @@ const INTERVALS = {
  * @constructor
  * @param {!String} range String formatted as an IS0 8601 time interval
  */
+
+
 function DateRange(start, end, options) {
-  var parts;
-  var s = start;
-  var e = end;
 
   if (options === undefined && ( (typeof end === 'object') && !(end instanceof moment || end instanceof Date) )) {
     options = end;
     e = end = undefined;
   }
 
-  if (arguments.length === 1 || end === undefined) {
-    if (typeof start === 'object' && start.length === 2) {
-      s = start[0];
-      e = start[1];
-    }
-    else if (typeof start === 'string') {
-      parts = start.split('/');
-      s = parts[0];
-      e = parts[1];
-    }
-  }
+  this.options = options = options || {};
+  
+  this._setRange = (options.contain) ? this._containIntersect : this._intersect;
+  
+  this.set(start, end, options);
+}
 
-  this.options = options || {};
-  this.start = (s === null) ? moment(-8640000000000000) : moment(s);
-  this.end   = (e === null) ? moment(8640000000000000) : moment(e);
+DateRange.prototype._setDuration = function() {
+  this.duration = moment.duration(this.end - this.start);
```

Comment:
since an actual range that does not intersect the limit range can case undefined ranges, this should be checking to see if start or end is undefined and returning undefined if so